earth_enterprise/src/fusion/gecombineterrain/gecombineterrain.cpp

```diff
@@ -232,28 +234,47 @@ int main(int argc, char **argv) {
                 &khGetopt::RangeValidator<uint32, 0, 1024>);
     options.opt("read_cache_block_size", read_cache_block_size,
                 &khGetopt::RangeValidator<uint32, 1, 1024>);
-
     if (!options.processAll(argc, argv, argn)) {
       usage(progname);
     }
     if (help) {
       usage(progname);
     }
+
     if (argn == argc) {
-      usage(progname, "No input indexes specified");
+      usage(progname, "No input indices specified");
     }
+    uint cmdDefaultCPUs = CommandlineNumCPUsDefault();
+    uint numavailable = GetMaxNumJobs();
+
+    /* 
+        Choosing number of CPUs:
+
+        numcpus: initially, the command line parameter --numcpus
+	
+        CommandLineNumCPUsDefault(): defined in /etc/opt/google/systemrc via maxjobs tag
```

Comment:
@tst-basilhuffman: Yes, I think, ideally, we should change this to a better name.  However, that's a minor improvement, so I'll approve even without it.